README.md

```diff
@@ -1,16 +1,56 @@
-# Apostrophe Boilerplate v2.x
+# CMS Example
 
-Apostrophe Boilerplate is a minimal starting point for [Apostrophe 2](https://github.com/punkave/apostrophe) projects.
+Based on Apostrophe Boilerplate v2.x, it's just a Node.js cms to use as sample project.
 
-To get started, we recommend taking a look at [our guide to creating your first project](http://apostrophecms.org/docs/tutorials/getting-started/creating-your-first-project.html). You could also take a look at [Apostrophe's CLI](https://github.com/punkave/apostrophe) or simply fork this repository.
+> Apostrophe Boilerplate is a minimal starting point for [Apostrophe 2](https://github.com/punkave/apostrophe) projects.
 
-Once you have a local copy of this project to work from, make sure to install its dependencies with `npm install`. With Apostrophe installed, the first thing to do create an admin user account so you're able to log into the CMS. Run the following command (this will prompt you for a password).
+# TL;DR;
 
 ```bash
-node app.js apostrophe-users:add admin admin
+$ curl -LO https://raw.githubusercontent.com/fjagugar/blog-example/master/docker-compose.yml
```

Comment:
Well, you actually need to download the repo in order to be able to run docker-compose right?

Reply:
You're right!